community-concepts/Interne taak/README.md

```diff
@@ -1,2 +1,7 @@
-# Internetaak schema
-Dit is het JSON-schema voor interne taak zoals deze in KISS (PodiumD Contact) geimplementeerd zit. 
+# Internetaak schema [VEROURDERD]
+Dit is het JSON-schema voor interne taak zoals deze in KISS (PodiumD Contact) geimplementeerd zat. 
```

Comment:
Is het nodig om expliciet applicatienamen zoals KISS en Open Klant te noemen hier? Ik zou zeggen dat Objecttypes onwetend moet blijven en geen enkele melding zou moeten maken van applicaties.